src/core.jsx

```diff
@@ -89,6 +76,7 @@ export default (
 
         // Load from props
         if (this.omitLoadInProps(this.props)) {
+          // eslint-disable-next-line react/destructuring-assignment
```

Comment:
Maybe you could disable this rule directly ?

Reply:
I prefer not to do this.
It reminds me that this is maybe bad :)

I try to update whenever this is possible without taking risk of breaking something.
The both times I didn't fix this are the times when I don't feel confortable